### ISSUE TITLE
cpan/IPC-SysV/SysV.xs: new build-time warnings

### ISSUE THREAD

**Issue by jkeenan** [Perl/perl5#17979]
At some point between dcd070d813 (May 23) and 23f1d933a1 (July 17?), new build-time warnings appeared in blead.  On an unthreaded build on linux I got this:
```
$ report-build-warnings 23f1d933a1.linux.unthreaded.maketestprep.output.txt.gz 
File:  23f1d933a1.linux.unthreaded.maketestprep.output.txt.gz

  Wsign-compare                              3

$ parse-build-warnings 23f1d933a1.linux.unthreaded.maketestprep.output.txt.gz 
File:  23f1d933a1.linux.unthreaded.maketestprep.output.txt.gz

[
  {
    char   => 24,
    group  => "Wsign-compare",
    line   => 382,
    source => "SysV.xs",
    text   => "comparison between signed and unsigned integer expressions",
  },
  {
    char   => 39,
    group  => "Wsign-compare",
    line   => 382,
    source => "SysV.xs",
    text   => "signed and unsigned type in conditional expression",
  },
  {
    char   => 11,
    group  => "Wsign-compare",
    line   => 384,
    source => "SysV.xs",
    text   => "comparison between signed and unsigned integer expressions",
  },
]
```
Most likely candidate:
```
commit 4fe28b115c7ca0681131f395e6f3709ef14ca309
Author:     Nicolas R <atoomic@cpan.org>
AuthorDate: Tue Jul 7 14:24:45 2020 -0600
Commit:     Nicolas R <nicolas@atoomic.org>
CommitDate: Fri Jul 17 15:32:22 2020 -0600

    Update IPC-SysV from 2.07 to 2.08

```
Since `cpan/IPC-SysV/` is maintained upstream on CPAN, this means we have to communicate with the maintainer.

@atoomic, can you investigate?

Thank you very much.
Jim Keenan

**Comment by atoomic** [comment]
note that upstream merged recently one of my PR for fixing `strict/warnings` 
view https://github.com/mhx/IPC-SysV/pull/7
2.08 was released https://metacpan.org/changes/distribution/IPC-SysV

The issues you have caught here seems to come from xs and should be addressed.
I wonder if these warnings are proper to this branch or not.

Going to investigate.

**Comment by atoomic** [comment]
we should move the discussion upstream https://github.com/mhx/IPC-SysV/issues/8

**Comment by atoomic** [comment]
should be fixed by https://github.com/mhx/IPC-SysV/pull/9 
we would need a new release once merge to avoid having some custom patches

**Comment by jkeenan** [comment]
Thanks to @mhx for new CPAN release of IPC-SysV (following https://github.com/mhx/IPC-SysV/pull/9).  Merged to blead in commit 2ce8ebb919.  Closing ticket.

Thank you very much.
Jim Keenan
